static/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -0,0 +1,45 @@
+# Caveats running DVC on Windows
+
+There can be few performance issues while running DVC on Windows by default. For
+some of these issues there are workarounds or configuration options available.
+Below are some of these issues detected so far and their workarounds. If you
+find any, please let us know!
+
+DVC in general is slower on Windows as compared to other platforms (like
+Linux, MacOS, etc.) that show pretty good performance. Upon investigation, it
+turns out that most of the delays are coming from [NTFS](https://en.wikipedia.org/wiki/NTFS)
+(New Technology File System) not being very good at handling directories with
+large number of files. NTFS is a file system that the Windows NT operating
+system uses for storing and retrieving files on a hard disk. [Here](https://superuser.com/questions/15192/bad-ntfs-performance)
+is the resource for reference. We can significantly improve NTFS performance by
+following workarounds:
+
+## Disable 8dot3 to improve the way NTFS manages the files
+
+With NTFS, user may want to disable 8dot3 as per
+[this](https://docs.microsoft.com/en-us/previous-versions/windows/it-pro/windows-server-2003/cc778996(v=ws.10))
+reference if he finds himself using over 300K files in a single directory. It
+enables NTFS to generate shorter file names and changes the way NTFS handles
+large files.
+
+## Whitelist in Microsoft Windows Defender
+
+Microsoft includes the Windows Defender antivirus program. If Windows Defender
```

Comment:
check also binary whitelisting, I believe DVC executable (or python) has to be whitelisted to speed up fs operations

Reply:
Yes, this is the general method of whitelisting any file or folder. We can whitelist DVC executable via this approach.

Reply:
it does not alert. It's just known that you can whitelist the DVC process to speedup things. It would be really great to give better instruction here - what file specifically we want to whitelist, for example. It's not straightforward.

Reply:
Done